src/shadowbox/model/access_key.ts

```diff
@@ -48,9 +48,20 @@ export interface AccessKey {
   readonly dataLimit?: DataLimit;
 }
 
+export interface AccessKeyCreateParams {
+  // The encryption method to use for the access key.
```

Comment:
Can you add the id as well?
That will allow people to export/import keys, or create keys for a user on demand, in a way that preserve the id used in the Prometheus metrics. That's a popular request from server operators.

In the implementation, make sure the id doesn't already exist.

Reply:
We can leave the id out of this PR.